crates/wysiwyg/src/dom/nodes/container_node.rs

```diff
@@ -765,6 +766,61 @@ where
     }
 }
 
+impl<S> ToPlainText<S> for ContainerNode<S>
```

Comment:
This implementation is not meant to be perfect, but good enough to preserve the flow of content by ensuring there are line breaks between block nodes, list items etc.